src/data/customer/Order.js

```diff
@@ -771,6 +780,7 @@ export const cancelBatchFileProcessAsync = async (batchId, token) => {
  * @param {string} filterObject {pickupDate, routeStatusIds, includeOrders, limit, offset}
  * @return {object} Promise resolve/reject
  */
+ //TODO: function should be moved to file customer/Job.js
```

Comment:
@vinay3206 Please evaluate effort required from your end

Reply:
@ccwukong @maartenhemmes  Change of the path in terms of Frontend Change